plugins/modules/inventory_intent.py

```diff
@@ -1228,8 +1233,8 @@ def resync_devices(self):
                         self.status = "success"
                         self.result['changed'] = True
                         self.result['response'] = execution_details
-                        self.log("Device Resynced Successfully and Resynced devices are :" + str(input_device_ips))
-                        self.msg = "Device " + str(input_device_ips) + " Resynced Successfully !!"
+                        self.msg = "Device Resynced Successfully and Resynced devices are: {0}".format(str(input_device_ips))
```

Comment:
Use caps for every words only if required. 
`Devices have been successfully resynced.  Devices resynced: {0}".format(str(input_device_ips))`

Reply:
updated.